nnpdf_data/nnpdf_data/new_commondata/STAR_2012_1JET_200GEV/filter.py

```diff
@@ -0,0 +1,115 @@
+import glob
+import sys
+
+import pandas as pd
+import yaml
+
+sys.path.append('../')
+from ATLAS_TTBAR_13TEV_HADR_DIF.utils import symmetrize_errors
```

Comment:
maybe you can morve this to the folder called `filter_utils` and create a module called `uncertainties` and place inside both and the compute covmat function from heremes

Reply:
done in 9c76548